tests/network-tests/src/fixtures/referendum/voteFixture.ts

```diff
@@ -0,0 +1,66 @@
+import { SubmittableExtrinsic } from '@polkadot/api/types'
+import { ISubmittableResult } from '@polkadot/types/types'
+import BN from 'bn.js'
+import { assert } from 'chai'
+import { Api } from '../../Api'
+import { StandardizedFixture } from '../../Fixture'
+import { QueryNodeApi } from '../../QueryNodeApi'
+import { EventDetails, AnyQueryNodeEvent } from '../../types'
+
+export type VotingParams = {
+  commitment: string
+  stake: BN
+}
+
+export class VoteFixture extends StandardizedFixture {
+  private votes: Map<string, VotingParams>
+  private failureExpected: boolean
+
+  constructor(api: Api, query: QueryNodeApi, votes: Map<string, VotingParams>, failureExpected = false) {
+    super(api, query)
+    this.votes = votes
+    this.failureExpected = failureExpected
+  }
+
+  protected getSignerAccountOrAccounts(): Promise<string | string[]> {
+    return Promise.resolve(Array.from(this.votes.keys()))
+  }
+
+  protected getExtrinsics(): Promise<
+    SubmittableExtrinsic<'promise', ISubmittableResult>[] | SubmittableExtrinsic<'promise', ISubmittableResult>[][]
+  > {
+    const results = Array.from(this.votes).map(async ([, params]) =>
+      this.api.tx.referendum.vote(params.commitment, params.stake)
+    )
+    return Promise.all(results)
+  }
+
+  protected getEventFromResult(result: ISubmittableResult): Promise<EventDetails<unknown>> {
+    return this.api.getEventDetails(result, 'referendum', 'VoteCast')
+  }
+
+  protected assertQueryNodeEventIsValid(qEvent: AnyQueryNodeEvent, i: number): void {
+    // TODO: implement QN checks
+  }
+
+  // overwriting for using decremental tips
```

Comment:
There was a comment explaining why it was needed in the council fixture, ie.:
```
    // Due to the fact that we need the transactions to be processed in the expected order
    // (which is not guaranteed by the nonce, because we're using different voter accounts),
    // we'll be including a small, decremental tip (10 JOY * (votersStakingAccounts.length - 1 - accIndex))
```
Could you add it here?

Also:
- It would probably be better to make `StandardizedFixture` have a simple config (like a flag for example) to enable/disable decremental tips
- `10` is a very small amount, I think it was used back when we had no fees & token decimals at all. `10 JOY` (`100_000_000_000`) would be more reasonable I think

Reply:
Is it ok if I update all the balances amount (`10 -> 10_000_000_000`) in a future PR, I can open a issue with a a tech-debt, maybe